services/dy-3dvis/simcoreparaviewweb/requirements.txt

```diff
@@ -0,0 +1,6 @@
+minio==4.0.0
+networkx==2.1
+psycopg2-binary==2.7.4
```

Comment:
I remember having read that you should not use the psycopg2-binary but build from source instead, i.e. get the postgresql-dev packages from distro.

Reply:
ok I'll have a look into this... I think I actually copy/pasted this from your stuff some time ago.. did you change that?